src/table/components/head/MenuList/MenuGroup.tsx

```diff
@@ -27,10 +27,66 @@ export const interceptClickOnMenuItems = (menuGroups: MenuItemGroup[], cache: Su
 type SubMenusOpenStatusCache = Record<string, React.Dispatch<React.SetStateAction<boolean>>>;
 let subMenusOpenStatusCache: SubMenusOpenStatusCache = {};
 
-const MenuGroupItems = ({ id, onClick, itemTitle, icon, enabled, subMenus }: HeadCellMenuItem) => {
+const MenuGroupItems = ({ autoFocus, id, onClick, itemTitle, icon, enabled, subMenus }: HeadCellMenuItem) => {
   const [openMenu, setOpenMenu] = useState(false);
   const anchorRef = useRef<HTMLDivElement | null>(null);
 
+  const moveToNextFocusItem = (currentFocus: Element) => {
+    let nextItem = currentFocus.nextElementSibling as HTMLDivElement;
+    if (!nextItem) {
+      nextItem = currentFocus.parentElement?.children[0] as HTMLDivElement;
+    } else if (nextItem.tagName === 'HR') {
+      nextItem = moveToNextFocusItem(nextItem);
+    }
+    return nextItem;
+  };
+
+  const moveToPreviousFocusItem = (currentFocus: Element) => {
+    let previousItem = currentFocus.previousElementSibling as HTMLDivElement;
+    if (!previousItem) {
+      const menuItemAmount = currentFocus?.parentElement?.children.length;
+      (currentFocus?.parentElement?.children[(menuItemAmount as number) - 1] as HTMLDivElement).focus();
+    } else if (previousItem.tagName === 'HR') {
+      previousItem = moveToPreviousFocusItem(previousItem);
+    }
+    return previousItem;
+  };
+
+  const handleKeyDown = (event: React.KeyboardEvent<HTMLLIElement>) => {
+    const { key } = event;
+
+    const currentFocus = document.activeElement as Element;
+
+    // The rest key are handled by handleKeyDown in MUIMenuList
+    if (key === 'ArrowDown') {
+      // Prevent scroll of the page
+      event.preventDefault();
+      // Stop triggering handleKeyDown in MUIMenuList
+      event.stopPropagation();
```

Comment:
So MUIMenuList has a key handler that you can use to navigate the list right? Why can't we use it?